bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -85,17 +85,16 @@
         let(:dependency_files) { bundler_project_dependency_files("subdependency") }
         its([:version]) { is_expected.to eq(Gem::Version.new("0.7.0")) }
 
-        # TODO: https://github.com/dependabot/dependabot-core/issues/2364
-        # context "that will be removed if other sub-dependencies are updated" do
-        #   let(:gemfile_fixture_name) { "subdependency_change" }
-        #   let(:lockfile_fixture_name) { "subdependency_change.lock" }
-        #   let(:dependency_name) { "nokogiri" }
-        #   let(:requirements) { [] }
-
-        #   pending "is updated" do
-        #     expect(subject.version).to eq(Gem::Version.new("1.10.9"))
-        #   end
-        # end
+        context "that will be removed if other sub-dependencies are updated" do
+          let(:gemfile_fixture_name) { "subdependency_change" }
+          let(:lockfile_fixture_name) { "subdependency_change.lock" }
+          let(:dependency_name) { "nokogiri" }
+          let(:requirements) { [] }
+
+          pending "is updated" do
```

Comment:
Well, I didn't reenable anything since the specs are still pending 🤣, let me see if they actually pass now.

Reply:
Alright, specs are still failing but I think it still has some value to actually run them but mark them as pending, so I will keep this PR with that goal.